README.md

```diff
@@ -313,36 +342,35 @@ build time. For example, the preview mode banner links to the git branch that
 the site was built from (when that environment variable is available at build
 time).
 
-## Ways to view the site at different stages
+## Ways to view the site at different stages 🔍
 
-Throughout the develop-deploy process there are a number of ways to view the
+Throughout the develop-deploy process, there are several ways to view the
 website. The following table summarizes the important ways in which those views
 differ from each other.
 
 | Name                       | How to access          | GitHub branch | Vercel env | Storyblok API key | Next.js preview? | Storyblok version param | Display top banner? | Top banner color | Button to enter/exit preview? |
 | -------------------------- | ---------------------- | ------------- | ---------- | ----------------- | ---------------- | ----------------------- | ------------------- | ---------------- | ----------------------------- |
 | Production                 | .com/.org URL          | `main`        | production | public            | off              | `published`             | No                  | n/a              | n/a                           |
-| Storyblok (yellow banner)  | via Storyblok UI       | `develop`     | preview    | preview           | on               | `draft`                 | Yes                 | yellow           | No                            |
```

Comment:
@trallard I think the breakage to this table needs to be fixed before this PR is merged, but otherwise yes, this PR is good to go. Agreed, the workflow updates should be in new PR(s).

Reply:
Working on the fix now.